packages/vue-renderer/src/renderers/ssr.js

```diff
@@ -39,7 +39,10 @@ export default class SSRRenderer extends BaseRenderer {
     )
   }
 
-  async devRenderToString(renderContext) {
+  creatSsrLog() {
+    if (!this.options.render.ssrLog) {
+      return { finish() {} }
```

Comment:
What if we just `return` here and later check for logger existence to finish it? This can prevent creating an extra closure for each SSR request in prod..

Reply:
Anyway, just a side suggestion. Probably this overhead is almost 0 ;)

Reply:
Make sense, what about now ?

Reply:
Seems good :)